lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -225,10 +239,11 @@ static FModuleOp checkSubModuleOp(FModuleOp topModuleOp, Operation *oldOp) {
 /// All standard expressions and handshake elastic components will be converted
 /// to a FIRRTL sub-module and be instantiated in the top-module.
 static FModuleOp createSubModuleOp(FModuleOp topModuleOp, Operation *oldOp,
+                                   bool hasClock,
                                    ConversionPatternRewriter &rewriter) {
   rewriter.setInsertionPoint(topModuleOp);
   using ModulePort = std::pair<StringAttr, FIRRTLType>;
-  llvm::SmallVector<ModulePort, 8> ports;
+  std::vector<ModulePort> ports;
```

Comment:
Why?  My understanding is that SmallVector is more memory efficient in this case?

Reply:
Oh my fault, this will be fixed.